crates/db_schema/src/schema.rs

```diff
@@ -390,6 +390,8 @@ diesel::table! {
         password_encrypted -> Text,
         email -> Nullable<Text>,
         show_nsfw -> Bool,
+        blur_nsfw -> Bool,
+        auto_expand -> Bool,
```

Comment:
These need to be added at the end, because they need to match the column order.

Reply:
Thanks, I've shifted them to appear in the order of my db migrations (`infinite_scroll_enabled` was added after mine)
The schema stuff seems to pass now 👍